### ISSUE TITLE
Crash after upgrading to v1.6.1

### ISSUE THREAD

**Issue by bes** [getsentry/sentry-webpack-plugin#85]
After I upgraded to `@sentry/webpack-plugin` v1.6.1 and `webpack` v4.19.1 I am seeing the following error:

```
TypeError: Cannot read property 'loader' of undefined
    at input.loaders.sillyClone.map.x (/Users/x/repos/spiideo/web-monolith/node_modules/@sentry/webpack-plugin/src/index.js:256:39)
    at Array.map (<anonymous>)
    at SentryCliPlugin.injectReleaseWithDebug (/Users/x/repos/spiideo/web-monolith/node_modules/@sentry/webpack-plugin/src/index.js:256:9)
    at SentryCliPlugin.apply (/Users/x/repos/spiideo/web-monolith/node_modules/@sentry/webpack-plugin/src/index.js:307:12)
    at webpack (/Users/x/repos/spiideo/web-monolith/node_modules/webpack/lib/webpack.js:37:12)
    at processOptions (/Users/x/repos/spiideo/web-monolith/node_modules/webpack-cli/bin/cli.js:441:16)
    at yargs.parse (/Users/x/repos/spiideo/web-monolith/node_modules/webpack-cli/bin/cli.js:528:3)
    at Object.parse (/Users/x/repos/spiideo/web-monolith/node_modules/yargs/yargs.js:563:18)
    at /Users/x/repos/spiideo/web-monolith/node_modules/webpack-cli/bin/cli.js:219:8
    at Object.<anonymous> (/Users/x/repos/spiideo/web-monolith/node_modules/webpack-cli/bin/cli.js:530:3)
    at Module._compile (internal/modules/cjs/loader.js:654:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)
    at Module.load (internal/modules/cjs/loader.js:566:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:506:12)
    at Function.Module._load (internal/modules/cjs/loader.js:498:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:695:10)
    at startup (internal/bootstrap/node.js:201:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:516:3)
```

We have a fairly complex (scripted / function based) webpack config, so I don't think it would be suitable to paste it verbatim, but if you want to see some part of it I would be happy to add it.

I see that the code at the offending line `@sentry/webpack-plugin/src/index.js:256:39` is relatively new (14 days or so).

Any idea of what this could be?

Thanks!

**Comment by gnclmorais** [comment]
I’m seeing the same behaviour on `webpack 3.12.0`.

**Comment by kamilogorek** [comment]
Hey, sorry for the inconvenience – fixed in https://github.com/getsentry/sentry-webpack-plugin/pull/86
For now, you can disable `debug` mode in the config and re-enable it once we release a patched version. Cheers!

**Comment by schettino** [comment]
Any news about this release? It's been merged already

**Comment by PeterMK85** [comment]
Same question here, release needed with the fixed `debug` option

**Comment by kamilogorek** [comment]
Sorry it took so long... released as `1.6.2`.
